lib/streams.gi

```diff
@@ -642,7 +642,7 @@ function( str )
     if fid = fail  then
         return fail;
     else
-        AddSet( InputTextFileStillOpen, fid );
+        ADD_SET( InputTextFileStillOpen, fid );
```

Comment:
Why?

Reply:
Because `AddSet` is not defined yet, and so the choices are:
 * Figure out how to move dependencies around to make `AddSet` available (which I gave up on after two attempts).
 * Move the `InstallValue` calls below to after when `AddSet` is defined, which might be acceptable, but then I introduce a new file which just installs these three values, which seems a bit overboard.

Reply:
OK, fair enough. I am just confused as to why this worked before?

Reply:
Because the functions calling `AddSet` weren't called inside this file: the function `AddSet` is declared but not defined yet. Adding the `StandardOutput`, `StandardInput`, `StandardError` streams here causes them to be called.

Now one *could* argue that one should move the installation of these streams to later, but that leaves the question of an appropriate place (see point 2 above).

Reply:
I am certainly not going to argue that; I merely wanted to understand what's going on. Thank you for explaining :-).